### PR TITLE
nginx: add headers_more

### PR DESCRIPTION
**Pull request by kulikov-a**
Hi!
can we add nifty headers_more module in nginx package please?)
thanks!

### CODE REVIEW

File: config/22.1/make.conf [hunk 1]
```diff
@@ -96,7 +96,7 @@ security_stunnel_SET=		TLS_LOG_IDENT
 security_suricata_SET=		HYPERSCAN NSS
 sysutils_msktutil_SET=		${PRODUCT_GSSAPI}
 www_lighttpd_UNSET=		LUA
-www_nginx_SET=			BROTLI NAXSI MAIL_IMAP MAIL_POP3 NJS VTS
+www_nginx_SET=			BROTLI NAXSI MAIL_IMAP MAIL_POP3 NJS VTS HEADERS_MORE
```
Comment (fichtner) [inline config/22.1/make.conf:99]: Can you sort this in alphabetically please?

Comment (kulikov-a) [reply config/22.1/make.conf]: done, sorry

**Comment by fichtner** [comment]
Merged, thanks

**Comment by kulikov-a** [comment]
Thanks!

**Pull request merged**